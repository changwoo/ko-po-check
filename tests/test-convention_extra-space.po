# -*-text-*-
#
msgid ""
msgstr ""
"Project-Id-Version: test\n"
"POT-Creation-Date: 2002-07-24 22:09+0900\n"
"PO-Revision-Date: 2002-07-24 22:09+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# basic
msgid "one"
msgstr "하나 "

# 이런 경우도 콜론 뒤에 공백 불필요
msgid "Alarms after"
msgstr "다음 시간이 지난 후에 알람: "

