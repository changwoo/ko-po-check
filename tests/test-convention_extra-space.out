-:16: 끝에 불필요하게 공백 문자가 있습니다
-:20: 끝에 불필요하게 공백 문자가 있습니다
